okhttp/src/test/java/okhttp3/internal/http2/HttpOverHttp2Test.kt

```diff
@@ -164,10 +162,13 @@ class HttpOverHttp2Test {
           .build()
       scheme = "http"
     }
-    cache = Cache(tempDir, Int.MAX_VALUE.toLong())
   }
 
   @AfterEach fun tearDown() {
+//    TODO reenable after https://github.com/square/okhttp/issues/8206
+//    fileSystem.checkNoOpenFiles()
```

Comment:
You can’t leave a `TODO: re-enable after X` when X is already closed; that’s cheating!

Reply:
https://github.com/square/okhttp/issues/8230